src/reducers/controls.ts

```diff
@@ -36,11 +40,24 @@ export interface MeasurementsControlState {
 
 export interface ControlsState extends BasicControlsState, MeasurementsControlState {}
 
+interface Defaults {
+  distanceMeasure: string
+  layout: Layout
+  geoResolution: string
+  filters: Record<string, any>
+  filtersInFooter: string[]
+  colorBy: string
+  selectedBranchLabel: string
+  tipLabelKey: typeof strainSymbol
```

Comment:
what's the advantage of doing this versus adding a proper type in `../utils/globals` and exporting it from there?

Reply:
I didn't want to dig a deeper hole and convert `src/utils/globals.js` to TypeScript. Even if I were to do that, I'm not sure that adding a proper type would be useful in this case. In `globals.ts` it would simply be:

```diff
- export const strainSymbol = Symbol('strain');
+ export const strainSymbol: unique symbol = Symbol('strain');
```

and the same `typeof strainSymbol` would still be used here.